org.firestormviewer.FirestormViewer.json

```diff
@@ -0,0 +1,89 @@
+{
+  "app-id":"org.firestormviewer.FirestormViewer",
+  "branch":"stable",
```

Comment:
Remove it or use `default-branch` instead.

Reply:
Removed.